components/Breadcrumbs.tsx

```diff
@@ -0,0 +1,27 @@
+import Head from "next/head";
+import Link from 'next/link';
+
+interface BreadcrumbsProps {
+    breadcrumbs: BreadcrumbItem[];
+}
+interface BreadcrumbItem {
+    href?: string;
+    title: string;
+}
+export default function Breadcrumbs({breadcrumbs}: BreadcrumbsProps) {
+    return (
```

Comment:
2 space indentering (her og andre steder)? 🙏 Tar selvkritikk for dårlig linting-regler.

Reply:
Breadcrumbs så heller ikke sånn meeega-bra ut på mobil, men det er ikke noen big deal for min del sånn i første iterasjon, det er sånne ting vi kan iterere på IMO